### PR TITLE
remoteDisconnect feature.

### PR DESCRIPTION
**Pull request by overflowz**
Please, check carefully before merging, because I'm not sure if I did it right. Thanks!

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -259,6 +260,23 @@ function adapter(uri, opts) {
         socket.leave(request.room, sendAck);
         break;
 
+      case requestTypes.remoteDisconnect:
+
+        var socket = this.nsp.connected[request.sid];
+        if (!socket) { return; }
+
+        function sendAck(){
```
Comment (darrachequesne) [inline index.js]: The sendAck function is useless here, isn't it?

**Comment by darrachequesne** [comment]
Looks good to me! Could you please add a test, and a line in the README?

**Comment by overflowz** [comment]
I'm kinda new to git(hub) but I'll try.

File: index.js (hunk 1 above)
Comment (darrachequesne) [inline index.js]: Sorry, my comment wasn't clear enough! What I thought of was:
```js
case requestTypes.remoteDisconnect:

  var socket = this.nsp.connected[request.sid];
  if (!socket) { return; }
 
  socket.disconnect(request.close);
  // the acknowledgement is still needed
  var response = JSON.stringify({
    requestid: request.requestid
  });
  pub.publish(self.responseChannel, response);
  break;
```

Comment (overflowz) [inline index.js]: Hey, could you please check it again?

**Comment by darrachequesne** [comment]
Thanks for that pull request! I added a test, to prevent future regressions.

**Comment by overflowz** [comment]
You're welcome!

**Pull request merged**